cpp/array.cpp

```diff
@@ -0,0 +1,31 @@
+// K'th smallest/ Largest Element in unsorted array
```

Comment:
@KadhamVj23 rename file according to the algorithm that is being implemented. `array.cpp` does not convey the work the program is doing.